custom/icds_reports/ucr/data_sources/dashboard/complementary_feeding_forms.json

```diff
@@ -65,23 +65,29 @@
             "location_id": {
               "type": "icds_user_location",
               "user_id_expression": {
-                "expression": {
-                  "datatype": null,
-                  "type": "property_path",
-                  "property_path": [
-                    "form",
-                    "meta",
-                    "userID"
-                  ]
-                },
-                "type": "root_doc"
+                "type": "named",
+                "name": "user_location_id"
               }
             },
             "location_type": "state"
           }
         },
```

Comment:
could further simplify this by removing the `nested` expression and using `location_property`

Reply:
Done